### PR TITLE
Scion ship names.

### PR DESCRIPTION
**Pull request by Benemuel**
Names for Scion ships for the busy space organizations.

### CODE REVIEW

File: code/modules/busy_space/organizations_vr.dm [hunk 1]
```diff
@@ -224,3 +224,32 @@
 						"a settlement needing our help",
 						"Forward Base Sigma-Alpha in ArCon space")
 
+/datum/lore/organization/gov/Scion
+    name = "Scion Imperium"
+    short_name = "Scion"
+    desc = "A Imperium of Desert Elves, slowly expanding and exploring the galaxy on there own terms instead of those of others./
```
Comment (Screemonster) [inline code/modules/busy_space/organizations_vr.dm]: A few typos, mostly there/there mixups in the description. Should read "on their own terms", "Their ships are..." and "Despite their lack of ships, their ground forces..."
History section misspells "calendar".

Comment (Benemuel) [inline code/modules/busy_space/organizations_vr.dm]: And this is why autocorrect is useful for me >_>

**Comment by Benemuel** [comment]
Also I can't tell for the life of me why it is failing the check.

**Comment by SpadesNeil** [comment]
If the Scion is just one planet with a couple of military ships I don't see the point in adding this, to be honest. I'd rather not, or everyone will want their own custom races and factions to be included.

**Comment by Benemuel** [comment]
Righto.

**Pull request closed without merging**